### PR TITLE
use isVisible instead of tooltipIsVisible

### PR DESCRIPTION
**Pull request by andrew-zenefits**

### CODE REVIEW

File: addon/components/tooltip-and-popover.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default EmberTetherComponent.extend({
   showOn: null,
   spacing: 10,
   tabindex: '0', // A positive integer (to enable) or -1 (to disable)
-  tooltipIsVisible: false,
+  isVisible: false,
```
Comment (MiguelMadero) [inline addon/components/tooltip-and-popover.js]: This is a breaking change, can we add an `Ember.computed.alias` to the tooltips so they maintain the `tooltipIsVisible` property? That way we can send a PR for this without having to do another major version bump. We can mark `tooltipIsVisible` as deprecated and remove in 3.0.

Comment (MiguelMadero) [inline addon/components/tooltip-and-popover.js]: Ok a computed.alias won't work, you will need to manually create a prop with getter and setter to handle it there so we don't override it.

**Comment by zen-tmq** [comment]
I don't think we should merge this commit. Is the reason behind this to maintain our own style? This is a third party library and should not subject to our style guides. Remember the goal is to remove our fork and use the main repro, unnecessary hacks will make this more difficult

**Comment by andrew-zenefits** [comment]
I am going to direct all of my ember-popover changes towards `sir-dunxalot/ember-tooltips`.

**Comment by MiguelMadero** [comment]
Quinn, the reason for this change is because we now have popover and
tooltips, so the old name no longer makes sense. I think we should live
with the old name in our codebase until origin has popovers and we add the
new prop there and an alias for backwards compatibility.

**Pull request closed without merging**